lib/mixin/vpp.py

```diff
@@ -164,10 +164,10 @@ def check_stack(self):
       params.update(width = self.cols * self.tilew, height = self.rows * self.tileh)
       params.update(reference = format_value(self.reference, **params))
     elif self.stack in ["hstack"]:
-      params.update(metric = dict(type = "md5"))
+      params.update(metric = dict(type = "ssim", miny = 0.97, minu = 0.97, minv = 0.97))
```

Comment:
```suggestion
      params.setdefault(metric = "md5")
```

Reply:
Okay, I understand what you mean. I'm trying to make changes now,thanks.